packages/frontend-v2/utils/theme/index.ts

```diff
@@ -72,7 +93,10 @@ const breakpoints = {
  * either direction (5xs, ..., 5xl) as needed.
  */
 const space = {
-  "2xs": "0.25rem",
+  "5xs": "0.05rem",
```

Comment:
:0 is all of this linda's design system stuff? :DD

Reply:
oop not really, just values I needed to follow linda's design. i don't think linda's designs follow the design system just yet, once they do we can update it :))